api/libproject/devstart.rb

```diff
@@ -321,6 +350,12 @@ def create_db_creds(args)
 })
 
 Common.register_command({
+  :invocation => "run-api",
+  :description => "Runs the api server (assumes database and config are already up-to-date.)",
```

Comment:
Is there a command (like `update-cloud-config`) to update the db? Or is that just automatically done with `dev-up`?

Reply:
dev-up now runs "docker-compose update-config", which updates the local configuration.

We could have a separate Ruby target for it but it's probably not necessary.